contracts/contracts/vault/VaultInitializer.sol

```diff
@@ -33,5 +33,7 @@ contract VaultInitializer is VaultStorage {
         autoAllocateThreshold = 25000e18;
         // Threshold for rebasing
         rebaseThreshold = 1000e18;
+        // Initialize all strategies
+        allStrategies = new address[](0);
```

Comment:
@DanielVF added this to make slither happy (uninitialised `allStrategies` variable). This is probably OK since the vault has already been initialised right?